frontends/rtlil/rtlil_parser.y

```diff
@@ -187,7 +187,11 @@ wire_stmt:
 
 wire_options:
 	wire_options TOK_WIDTH TOK_INT {
-		current_wire->width = $3;
+		if ($3 > 0x1000000) {
```

Comment:
it maybe better to use a constexpr defined in rtlil.h instead of repeating the magic number everywhere; my 2-cents